### PR TITLE
552 fix val loss typo

### LINKED ISSUE #552: Bug in Self Supervised Pretraining
During pretraining the validation loss is not correctly calculated. 
https://github.com/Project-MONAI/tutorials/blob/e1088488c1bcd9c53eaebcdc8b727fe98c2463b9/self_supervised_pretraining/ssl_script_train.py#L211

I belive it should be 
`total_val_loss /= val_step`

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

Fixes #552 .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Review by wyli — APPROVED** [review]
Thanks, it looks good to me.
